setup/generate_usptream_rust_keylime_code_coverage/test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+# for Packit PRs we would be uploading code coverage files unless forbidden
+[ -n "${PACKIT_SOURCE_URL}" -a -z "${UPLOAD_COVERAGE}" ] && UPLOAD_COVERAGE=1
+UPLOAD_URL=https://transfer.sh
+
+
+rlJournalStart
+
+    rlPhaseStartSetup "Collect code coverage for rust components"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        #delete coverage script, in dir are mandatory only coverage files
+        rlRun "pushd ${__INTERNAL_limeCoverageDir}"
+        # export in format for user
+        rlRun "grcov . --binary-path /usr/local/bin/keylime_agent -s . -t html --branch --ignore-not-existing -o coverage_html"
+        # export coverage report in format compatible for codecov
+        rlRun "grcov . --binary-path /usr/local/bin/keylime_agent -s . -t lcov --branch --ignore-not-existing -o rust_coverage.txt"
+        # create tar file for uploading coverage file
+        rlRun "tar --create --file rust_coverage.tar coverage_html"
+        rlFileSubmit  rust_coverage.tar
+        rlFileSubmit rust_coverage.txt
+        if [ "${UPLOAD_COVERAGE}" == "1" ]; then
+            # upload coverage.tar.gz
+            rlRun -s "curl --upload-file rust_coverage.tar $UPLOAD_URL"
+            URL=$( grep -o 'https:[^"]*' $rlRun_LOG )
+            rlLogInfo "HTML code coverage report is available as GZIP archive at $URL"
+            #upload coverage_rust.txt reports
+            rlRun -s "curl --upload-file rust_coverage.txt $UPLOAD_URL"
+            URL=$( grep -o 'https:[^"]*' $rlRun_LOG )
+            rlLogInfo "rust_coverage.txt report is available at $URL"
+            rlRun -s "curl --upload-file tarpaulin-report.tar $UPLOAD_URL"
+            URL=$( grep -o 'https:[^"]*' $rlRun_LOG )
+            rlLogInfo "HTML code coverage report is available as GZIP archive at $URL"
+            #upload coverage_rust.txt reports
+            rlRun -s "curl --upload-file cobertura.xml $UPLOAD_URL"
+            URL=$( grep -o 'https:[^"]*' $rlRun_LOG )
+            rlLogInfo "cobertura.xml report is available at $URL"
+        fi
+        rlRun "rm -rf ${__INTERNAL_limeCoverageDir}/*"
```

Comment:
Why do you delete the content here?

Reply:
When you run plan several times, you have in coverage dir a files from previous runs. So I decided to after each run wipe out.

Reply:
Yep. But TBH, I don't expect us to generate multiple different coverage reports in sequence. I think it is better to keep it in place in case you would like to work with it manually.
Also, with this approach you would delete also Python coverage data that are stored in the very same directory.